L125-python-absl-logging-support.md

```diff
@@ -0,0 +1,153 @@
+Title
+----
+* Author(s): sreenithi
+* Approver: sergiitk, asheshvidyut
+* Status: Draft
+* Implemented in: Python
+* Last updated: 17-09-2025
+* Discussion at: <google group thread> (filled after thread exists) xxxxxxx
+
+## Abstract
+
+This proposal describes the changes necessary to add absl logging support in
+gRPC Python hence solving the user warning:
+
+```sh
+WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
+```
+
+
+## Background
+
+gRPC Core changed its logging system to use absl logging as per
+[L117][L117].
+
+After this change, gRPC python users started seeing the warning log -
+`WARNING: All log messages before absl::InitializeLog() is called are written to STDERR`.
```

Comment:
How do we reproduce this? I was only able to get it printed with `export GRPC_VERBOSITY=INFO` (which also prints out `Log level INFO is not suitable for production. Prefer WARNING or ERROR. However if you see this message in a debug environment or test environment it is safe to ignore this message.`, or with `export GRPC_TRACE=connectivity_state` (or any other module).